modules/nodes-nodes-working-updating.adoc

```diff
@@ -7,6 +7,11 @@
 
 You can update any label on a node.
 
+[NOTE]
+====
+Node labels are not persisted after a node is deleted even if the node is backed up by a Machine.
```

Comment:
I might add something like, "If you need to ensure that the labels persist if the Machine is replaced, add the labels to the MachineSet that controls the Machine instead."